plugins/modules/zos_fetch.py

```diff
@@ -170,66 +171,66 @@
 
 RETURN = r"""
 file:
-    description: The source file path on remote machine
+    description: The source file path on the remote machine.
     returned: success
     type: str
     sample: SOME.DATA.SET
 dest:
-    description: The destination file path on controlling machine
+    description: The destination file path on the controlling machine.
     returned: success
     type: str
     sample: /tmp/SOME.DATA.SET
 is_binary:
-    description: Indicates which transfer mode was used to fetch the file
+    description: Indicates which transfer mode was used to fetch the file.
     returned: success
     type: bool
     sample: True
 checksum:
-    description: The SHA256 checksum of the fetched file
+    description: The SHA256 checksum of the fetched file.
     returned: success and src is a non-partitioned data set
     type: str
     sample: 8d320d5f68b048fc97559d771ede68b37a71e8374d1d678d96dcfa2b2da7a64e
 data_set_type:
-    description: Indicates the fetched file's data set type
+    description: Indicates the fetched file's data set type.
```

Comment:
@asifmahmud do we want to say " file's data set type" , trying to see were 'file's' fit into a data set , maybe just remove 'files' so it reads "Indicates the fetched data set type."?

Reply:
Replaced the description to Indicates the fetched data set type